src/contracts/mocks/ism.cairo

```diff
@@ -0,0 +1,28 @@
+#[starknet::contract]
+pub mod ism {
+    use alexandria_bytes::{Bytes, BytesTrait, BytesStore};
+    use hyperlane_starknet::contracts::libs::message::{Message, MessageTrait};
+    use hyperlane_starknet::interfaces::{
+        IInterchainSecurityModule, IInterchainSecurityModuleDispatcher,
+        IInterchainSecurityModuleDispatcherTrait, ModuleType
+    };
+    use starknet::ContractAddress;
+
+    #[storage]
+    struct Storage {}
+    #[abi(embed_v0)]
+    impl IMessageidMultisigIsmImpl of IInterchainSecurityModule<ContractState> {
+        fn module_type(self: @ContractState) -> ModuleType {
+            ModuleType::MESSAGE_ID_MULTISIG(starknet::get_contract_address())
+        }
+
+        fn verify(
+            self: @ContractState,
+            _metadata: Bytes,
+            _message: Message,
+            _validator_configuration: ContractAddress
```

Comment:
why this ?

Reply:
_validator_configuration refers to the contract defining the set of validators and the threshold conditions. The user can simply mock this contract and specify conditions on validator selections. https://docs.hyperlane.xyz/docs/reference/ISM/multisig-ISM-interface

Reply:
yes. but its not the spec, so we should remove this arg cc https://docs.hyperlane.xyz/docs/reference/ISM/specify-your-ISM